tests/scripts/thread-cert/Cert_5_5_05_SplitMergeREED.py

```diff
@@ -175,11 +183,71 @@ def test(self):
         self.assertEqual(self.nodes[ROUTER1].get_state(), 'child')
         self.assertEqual(self.nodes[REED1].get_state(), 'router')
 
+        self.collect_ipaddrs()
+
         addrs = self.nodes[LEADER].get_addrs()
         for addr in addrs:
             if addr[0:4] != 'fe80':
                 self.assertTrue(self.nodes[ROUTER1].ping(addr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        LEADER = pv.vars['LEADER']
+        ROUTER_1 = pv.vars['ROUTER_1']
+        ROUTER_2 = pv.vars['ROUTER_2']
+        ROUTER_3 = pv.vars['ROUTER_3']
+        REED = pv.vars['REED']
+        router1_pkts = pkts.filter_wpan_src64(ROUTER_1)
+        router2_pkts = pkts.filter_wpan_src64(ROUTER_2)
+        reed_pkts = pkts.filter_wpan_src64(REED)
+
+        router2_pkts.filter_mle_cmd(MLE_CHILD_ID_RESPONSE).must_next()
+        _start_idx = router2_pkts.index
+        _reed_pkts = reed_pkts.range(_start_idx)
+
+        # Step 3: Remove Router_3 from the network
+        # Router_1 attempt to re-attach to partition.
+        # Sends multicast Parent Request to Routers and REEDs
+        router1_pkts.range(_start_idx).must_next()
+        router1_pkts.filter_mle_cmd(MLE_PARENT_REQUEST).must_next()
+        _end_idx = router1_pkts.index
+
+        # Step 2: The DUT MUST NOT attempt to become an active router by sending an Address Solicit Request
+        reed_pkts.copy().range(
```

Comment:
The copy here is unnecessary since we are using `filter.must_not_next()`. So we can never move `reed_pkts`'s index here. 
```suggestion
        reed_pkts.range(
```

BTW, `pkts.range(..., cascade=False)` is equivalent as `pkts.copy().range(...)`.